src/WebJobs.Script/Environment/EnvironmentSettingNames.cs

```diff
@@ -30,6 +30,8 @@ public static class EnvironmentSettingNames
         public const string SkipSslValidation = "SCM_SKIP_SSL_VALIDATION";
         public const string EnvironmentNameKey = "AZURE_FUNCTIONS_ENVIRONMENT";
         public const string EasyAuthEnabled = "WEBSITE_AUTH_ENABLED";
+        public const string AzureWebJobsSecretStorageVaultName = "AzureWebJobsSecretStorageVaultName";
+        public const string AzureWebJobsSecretStorageAuthConnectionString = "AzureWebJobsSecretStorageAuthConnectionString";
```

Comment:
I think these setting names should be:

AzureWebJobsSecretStorageKeyVaultName
AzureWebJobsSecretStorageKeyVaultConnectionString